docs/advanced_usage.rst

```diff
@@ -34,6 +34,35 @@ hooks, as these can be run on any platform. However, if you intend for your
 template to only be run on a single platform, a shell script (or `.bat` file
 on Windows) can be a quicker alternative.
 
+.. note::
+    Make sure your hook scripts work in a robust manner. If a hook script fails
+    (that is, `if it finishes with a nonzero exit status
+    <https://docs.python.org/2/library/sys.html#sys.exit>`_), the project
```

Comment:
@eliasdorneles It would be nice to indicate the Python3 documentation instead of Legacy Python.